### DISCUSSION TITLE
Themed images aren't displayed in the mobile app

### DISCUSSION THREAD

**Discussion by saket** [community/community#81663] [Mobile]
Hello, the header image on my repository isn't displayed in your android app. Can you please check?

https://github.com/saket/telephoto

![Screenshot_20231224-012628.png](https://github.com/community/community/assets/2387680/564135a4-1e0c-4dfa-9603-d71dd3724a4e)

**Comment by kangarruu** [comment]
@saket thank you for taking the time to write in. I have added your report to the internal issue where this is being investigated. Please continue to check back for updates in future releases. Though I can't give you a definitive timeline for when this will be resolved, rest assured that this is on our radar!
